test/test-dtutils.lua

```diff
@@ -0,0 +1,116 @@
+--[[
+
+    test-dtutils.lua - the start of a testing framework idea....
+
+    copyright (c) 2016 Bill Ferguson
+
+    TODO: Think this out a little better
```

Comment:
I looked at luarocks and I think this is doable.  I'm coding the Nik plugin as a separate plugin, so I'll use it as the test case for installing a plugin from an external location using an embedded luarocks.